src/domain/safe/entities/__tests__/multisig-transaction.builder.ts

```diff
@@ -0,0 +1,50 @@
+import { MultisigTransaction } from '../multisig-transaction.entity';
+import { Operation } from '../operation.entity';
+import { faker } from '@faker-js/faker';
+import { Builder, IBuilder } from '../../../../__tests__/builder';
+import {
+  confirmationBuilder,
+  toJson as confirmationToJson,
+} from './multisig-transaction-confirmation.builder';
+
+export function multisigTransactionBuilder(): IBuilder<MultisigTransaction> {
+  return Builder.new<MultisigTransaction>()
+    .with('baseGas', faker.datatype.number())
+    .with('blockNumber', faker.datatype.number())
+    .with('confirmations', [confirmationBuilder().build()])
+    .with('confirmationsRequired', faker.datatype.number())
+    .with('data', faker.datatype.hexadecimal())
+    .with('dataDecoded', faker.datatype.json())
+    .with('ethGasPrice', faker.datatype.hexadecimal())
+    .with('executionDate', faker.date.recent())
+    .with('fee', faker.datatype.hexadecimal())
```

Comment:
executer wasn't included https://github.com/5afe/safe-client-gateway-nest/pull/222/files#diff-272990ad7b76ae9ca394cb353d1b93b87c6713630a48c8d1d5dd0b9a67590234L23

Reply:
Fixed in [b793a9d](https://github.com/5afe/safe-client-gateway-nest/pull/222/commits/b793a9d39a95f02911e8b2276a615103919fac24)